modules/ocf_irc/files/reload-ssl.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Log into the IRC server and reload the SSL cert.
+
+# Use a random nick to ensure no collisions with users on the server
+username=$(pwgen --secure --no-numerals 16 1)
```

Comment:
if I'm understanding this right, would it be possible to use the "ocf" admin user that appears to already exist according to inspircd.conf.erb?

Reply:
I'll rename this variable to "nick" so it's more clear. If the "admin" user is online, using the ocf nick, this script will fail. So we set our nick to something random, but our "user" to the proper name, so it'll always work.